src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -4,17 +4,34 @@ import {
     Text,
     TouchableOpacity,
     ActivityIndicator,
+    TextInput,
 } from 'react-native';
 import PropTypes from 'prop-types';
 import {withOnyx} from 'react-native-onyx';
 import ONYXKEYS from '../../../ONYXKEYS';
 import styles from '../../../styles/styles';
 import themeColors from '../../../styles/themes/default';
+import BigNumberPad from '../../../components/BigNumberPad';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../../components/withWindowDimensions';
```

Comment:
Is `windowDimensionsPropTypes` necessary here? Asking because the other Modals don't seem to need this.

Perhaps it was to fix the status bar overlap issue introduced by react-native-navigation? If so, the latest master commits should fix this.

Reply:
I'm using `isSmallScreenWidth` prop from `windowDimensionsPropTypes` to separate between mobile and web/desktop. So, I decide to show/hide `BigNumberPad`.